org.openstreetmap.josm.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: org.openstreetmap.josm
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: josm
+rename-icon: josm
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+    env:
+        JAVA_HOME: /usr/lib/sdk/openjdk11/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: josm
+    disabled: true
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/enable.sh
+      # - ant/bin/ant dist
+
+    sources:
+      - type: git
+        url:  git://github.com/openstreetmap/josm.git
```

Comment:
It has to point to tag or commit to be reproducible; please don't align keys and values like that.

Reply:
this module is currently disabled. See a few lines above.

Reply:
I think it's bad practice to have WIP code as part of the manifest as it will surely be used as example by someone else.

Reply:
I think it's okay. The difference is, of course, that you can exercise your powers to block progress here.
Do you?

Reply:
Too irrelevant to fight over this of course.